src/cablab/providers/albedo.py

```diff
@@ -33,15 +29,15 @@ def get_variable_descriptors(self):
         return {
             VAR_NAME_BRIGHT: {
                 'data_type': numpy.float32,
-                'fill_value': numpy.nan,
+                'fill_value': FILL_VALUE,
                 'units': '-',
```

Comment:
At the moment, it always uses the original fill_value. Do you think we should use a standardized fill_value for all data? I think it can be easily implemented, but we have to consider the case where the fill_value data type does not match with the data type of the actual data.